app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -185,7 +185,7 @@ Ensure that migrations have been run against the database prior to running the g
 
 ### Set up the zone control planes
 
-You need the following values to pass to each zone control plane setup:
+For every individual zone an own cluster is required. You need the following values to pass to each zone control plane setup:
```

Comment:
I think it should be under the Kubernetes tab

Reply:
@schogges if you've made the code change to fix this suggestion, just go ahread and mark this conversaion as "Resolved", don't need to wait for the author to come back and confirm.